src/allDrugs.jl

```diff
@@ -214,3 +214,51 @@ function avgRepsParams(rep1, rep2, rep3, concs)
     stds[:, :, 5] = palstd
     return avgs, stds
 end
+
+function optim_all(concs::Array{Float64, 2}, g1::Array{Float64, 3}, g2::Array{Float64, 3})
+    residuals(hillParams) = residHillAll(hillParams, concs, g1, g2)
+    lowPiece = [0.01, 3.0, 0.9, 0.0, 0.0]
+    low = vcat(
+        minimum(concs[:, 1]),
+        lowPiece,
+        minimum(concs[:, 2]),
+        lowPiece,
+        minimum(concs[:, 3]),
+        lowPiece,
+        minimum(concs[:, 4]),
+        lowPiece,
+        minimum(concs[:, 5]),
+        lowPiece,
+        1e-9,
+        1e-9,
+        0.45,
+        3,
+        15,
+        0,
+        0,
+    )
+    highPiece = [10.0, 7.0, 2.3, 3.0, 3.0]
+    high = vcat(
+        maximum(concs[:, 1]),
+        highPiece,
+        maximum(concs[:, 2]),
```

Comment:
Is this function what you meant?
Should I keep the notebook?

Reply:
Yep, this looks good.